content/hi/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,649 @@
+  पहला बेदखली अनुरोध।
+
+डिफ़ॉल्ट रूप से, नोड नियंत्रक प्रत्येक 5 सेकंड में प्रत्येक नोड की स्थिति की जाँच करता है।
+इस अवधि को `--node-monitor-period` फ्लैग का उपयोग करके कॉन्फ़िगर किया जा सकता है
+`kube-controller-manager` घटक।
+
+### बेदखली पर दर सीमा
+
+ ज्यादातर मामलों में, नोड नियंत्रक बेदखली दर को सीमित करता है
+`--node-eviction-rate' (डिफ़ॉल्ट 0.1) प्रति सेकंड, जिसका अर्थ है कि यह पॉड्स को बेदखल नहीं करेगा
+प्रति 10 सेकंड में 1 से अधिक नोड से।
+
+किसी दिए गए उपलब्धता क्षेत्र में नोड होने पर नोड बेदखली व्यवहार बदल जाता है
+अस्वस्थ हो जाता है। नोड नियंत्रक जाँचता है कि ज़ोन में कितने प्रतिशत नोड हैं
+अस्वास्थ्यकर हैं (`तैयार` स्थिति `अज्ञात` या `गलत` है) पर
+उसी समय:
+
+- यदि अस्वास्थ्यकर नोड्स का अंश कम से कम `--अस्वस्थ-क्षेत्र-दहलीज` है
+  (डिफ़ॉल्ट 0.55), तो निष्कासन दर कम हो जाती है।
+- यदि क्लस्टर छोटा है (यानी इससे कम या इसके बराबर है
+  `--लार्ज-क्लस्टर-साइज़-थ्रेशोल्ड' नोड्स - डिफ़ॉल्ट 50), फिर बेदखली रोक दी जाती है।
+- अन्यथा, बेदखली दर घटाकर `--द्वितीयक-नोड-बेदखली-दर' कर दी जाती है
+  (डिफ़ॉल्ट 0.01) प्रति सेकंड।
+
+उपलब्धता क्षेत्र के अनुसार इन नीतियों को लागू करने का कारण एक है
+उपलब्धता क्षेत्र नियंत्रण तल से विभाजित हो सकता है जबकि अन्य बने रहते हैं
+जुड़े हुए। यदि आपका क्लस्टर एकाधिक क्लाउड प्रदाता उपलब्धता क्षेत्रों तक नहीं फैला है,
+तब बेदखली तंत्र प्रति-ज़ोन अनुपलब्धता को ध्यान में नहीं रखता है।
+
+उपलब्धता क्षेत्रों में अपने नोड्स को फैलाने का एक प्रमुख कारण यह है कि
+जब एक पूरा क्षेत्र नीचे चला जाता है तो वर्कलोड को स्वस्थ क्षेत्रों में स्थानांतरित किया जा सकता है।
+इसलिए, यदि किसी क्षेत्र में सभी नोड अस्वस्थ हैं, तो नोड नियंत्रक पर बेदखल करता है
+`--node-eviction-rate` की सामान्य दर। कोने का मामला तब होता है जब सभी जोन होते हैं
+पूरी तरह से अस्वस्थ (क्लस्टर में कोई भी नोड स्वस्थ नहीं है)। ऐसे में
+मामला, नोड नियंत्रक मानता है कि कनेक्टिविटी के साथ कुछ समस्या है
+नियंत्रण तल और नोड्स के बीच, और कोई निष्कासन नहीं करता है।
+(यदि कोई आउटेज हो गया है और कुछ नोड्स फिर से दिखाई देते हैं, तो नोड नियंत्रक करता है
+पॉड्स को शेष नोड्स से बेदखल करें जो अस्वास्थ्यकर या अगम्य हैं)।
+
+नोड कंट्रोलर नोड्स पर चल रहे पॉड को बेदखल करने के लिए भी जिम्मेदार है
+`NoExecute` दागी, जब तक कि उन पॉड्स उस कलंक को बर्दाश्त नहीं करते।
+नोड नियंत्रक {{< glosary_tooltip text="taints" term_id="taint" >}} भी जोड़ता है
+नोड अगम्य या तैयार नहीं जैसी नोड समस्याओं के अनुरूप। इसका मतलब यह है
+कि अनुसूचक पॉड्स को अस्वास्थ्यकर नोड्स पर नहीं रखेगा।
+
+## संसाधन क्षमता ट्रैकिंग {#नोड-क्षमता}
+
+नोड ऑब्जेक्ट नोड की संसाधन क्षमता के बारे में जानकारी ट्रैक करते हैं: उदाहरण के लिए, राशि
+उपलब्ध मेमोरी की संख्या और CPU की संख्या।
+नोड्स जो [स्वयं पंजीकरण](#स्व-पंजीकरण-ऑफ-नोड्स) के दौरान अपनी क्षमता की रिपोर्ट करते हैं
+पंजीकरण। यदि आप [मैन्युअल रूप से](#मैन्युअल-नोड-प्रशासन) एक नोड जोड़ते हैं, तो
+जब आप इसे जोड़ते हैं तो आपको नोड की क्षमता की जानकारी सेट करने की आवश्यकता होती है।
+
+कुबेरनेट्स {{< glosary_tooltip text="scheduler" term_id="kube-scheduler" >}} यह सुनिश्चित करता है
```

Comment:
Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.